src/sentry/eventstream/kafka/protocol.py

```diff
@@ -211,6 +211,13 @@ def get_task_kwargs_for_message_from_headers(
                 )
             task_state["group_states"] = group_states
 
+            # default in case queue is not sent
+            task_state["queue"] = (
```

Comment:
wasn't sure if this should be in `task_state` or `event_state` but this feels more appropriate

Reply:
Yeah this sounds right to me, since we include the `group_states` here as well